sniproxy.cpp

```diff
@@ -153,8 +153,8 @@ SNIProxy::SNIProxy(xcb_window_t wid, QObject* parent):
     const uint32_t windowMoveConfigVals[2] = { 0, 0 };
 
     xcb_configure_window(c, wid,
-                             XCB_CONFIG_WINDOW_X | XCB_CONFIG_WINDOW_Y,
-                             windowMoveConfigVals);
```

Comment:
Sorry for this. It was late night...

I'll fix this and other your notes soon.